src/App.test.tsx

```diff
@@ -1,9 +1,15 @@
-import React from 'react';
-import { render, screen } from '@testing-library/react';
-import App from './App';
+import React from 'react'
+import { Provider } from 'react-redux'
+import { render, screen } from '@testing-library/react'
+import { store } from './store/store'
+import App from './App'
 
 test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
-});
+  render(
+    <Provider store={store}>
+      <App />
+    </Provider>,
+  )
+  const linkElement = screen.getByText(/click/i)
+  expect(linkElement).toBeInTheDocument()
+})
```

Comment:
You might wanna add a test that gives mocked response from the API consumed in App and check if expected elements are shown. And make sure to add tests for each component too as you might be able to cache side effects in other changes. Even snapshots can be sufficient for them.